Shared/RollingFileLogger.swift

```diff
@@ -6,16 +6,16 @@ import Foundation
 import XCGLogger
 
 //// A rolling file logger that saves to a different log file based on given timestamp.
-public class RollingFileLogger: XCGLogger {
+open class RollingFileLogger: XCGLogger {
 
-    private static let TwoMBsInBytes: Int64 = 2 * 100000
-    private let sizeLimit: Int64
-    private let logDirectoryPath: String?
+    fileprivate static let TwoMBsInBytes: Int64 = 2 * 100000
+    fileprivate let sizeLimit: Int64
+    fileprivate let logDirectoryPath: String?
 
     let fileLogIdentifierPrefix = "com.mozilla.firefox.filelogger."
 
-    private static let DateFormatter: NSDateFormatter = {
-        let formatter = NSDateFormatter()
+    fileprivate static let DateFormatter: Foundation.DateFormatter = {
```

Comment:
Can we refer to `DateFormatter` without the explicit call to `Foundation`? If not, perhaps importing `Foundation` at the top is required?